learn/getting_started/quick_start.md

```diff
@@ -1,111 +1,219 @@
+888 Y888P 888 d8P  Y8b 888 888 888     "Y88b. d8P  Y8b     "88b 888P"  d88P"    888 "88b
+888  Y8P  888 88888888 888 888 888       "888 88888888 .d888888 888    888      888  888
+888   "   888 Y8b.     888 888 888 Y88b  d88P Y8b.     888  888 888    Y88b.    888  888
+888       888  "Y8888  888 888 888  "Y8888P"   "Y8888  "Y888888 888     "Y8888P 888  888
 
-[API references](/reference/api/documents.md)
+Database path:       "./data.ms"
+Server listening on: "127.0.0.1:7700"
+```
 
-### Checking updates
+Please note that **if you're using a terminal, you will need to open a second window in addition the one running MeiliSearch and use that second window to send the curl commands**.
 
-Most actions in MeiliSearch are [asynchronous](/learn/advanced/asynchronous_updates.md), including the document addition process.
+## Step 2: Add documents
 
-Asynchronous actions return a JSON object that contains only an `updateId` attribute. This is a **successful response**, indicating that the operation has been taken into account, but may not have been executed yet.
+Now that you have [successfully launched a MeiliSearch instance](#step-1-setup-and-installation), the next step is adding some data to search through.
 
-You can check the status of the operation via the `updateId` and the [get update status route](/reference/api/updates.md). Checking the update status of an operation is never mandatory, but can prove useful in tracing the origin of errors or unexpected behavior.
+MeiliSearch stores data in the form of discrete records, called [documents](/learn/core_concepts/documents.md). Documents are grouped into large collections, called [indexes](/learn/core_concepts/indexes.md).
 
-You can also check the status of all updates for a given index:
+To create an index called `movies` and add documents there, use:
 
-<CodeSamples id="get_all_updates_1" />
+<CodeSamples id="getting_started_add_documents_md" />
 
-The response to this command will depend on the status of updates for your index. Here is an example response showing a document addition request waiting to be processed:
+After adding documents, you should receive a response like this:
 
 ```json
-[
-    {
-        "status": "enqueued",
-        "updateId": 0,
-        "type": {
-            "name": "DocumentsAddition",
-            "number": 30
-        },
-        "enqueuedAt": "2021-02-14T14:07:09.364505700Z"
-    }
-]
+{
+    "uid": 0,
+    "indexUid": "movies",
+    "status": "enqueued",
+    "type": "documentsAddition",
+    "enqueuedAt": "2021-08-11T09:25:53.000000Z"
+}
 ```
 
-See our guide on [asynchronous updates](/learn/advanced/asynchronous_updates.md) or the [updates API reference](/reference/api/updates.md) for more information.
+Like most database operations in MeiliSearch, document addition is [asynchronous](/learn/advanced/asynchronous_updates.md). This means that the operation will be added to a queue and processed when it reaches the front, rather than happening immediately.
```

Comment:
Do we want to show the user how the status will go from `enqueued` to `processing` to `succeeded` or `failed` at this stage? Issue #701 
It is mentioned in the Asynchronous operations page PR #1208